packages/veritone-widgets/src/widgets/EngineSelection/EngineListView/SelectBar/index.js

```diff
@@ -31,6 +32,10 @@ export default class SelectBar extends React.Component {
   };
 
   render() {
+    if (this.props.hideActionMenuItems) {
```

Comment:
seems like "hide action menu items" actually means "hide this entire component", in which case, this hiding logic should be in the parent component that renders this one.